src/io/flutter/inspector/WidgetPerfTipsPanel.java

```diff
@@ -146,6 +146,7 @@ private void updateTip() {
         }
       }
 
+      // TODO(devoncarew): We never query changedEditors.
```

Comment:
go ahead and delete. This is code that should have been removed in a refactor.

Reply:
done